gobblin-data-management/src/main/java/gobblin/data/management/copy/hive/HiveCopyEntityHelper.java

```diff
@@ -605,7 +605,7 @@ private int addSharedSteps(List<CopyEntity> copyEntities, String fileSet, int in
 
     DeleteFileCommitStep deleteStep = DeleteFileCommitStep.fromPaths(targetFs, diffPathSet.pathsToDelete,
         this.dataset.properties);
-    copyEntities.add(new PrePublishStep(fileSet, Maps.<String, Object>newHashMap(), deleteStep, stepPriority++));
+    copyEntities.add(new PrePublishStep(fileSet, Maps.<String, Object>newHashMap(), deleteStep, stepPriority));
```

Comment:
I see, it is probably because it is the last use of stepPriority in the method, so yes, it won't be used again, and nothing will break for now. The problem is if later we add new steps, we will probably forget to change this line back. Is it easy to tell it to ignore this warning?